tests/feature/gherkin_terminal_reporter.feature

```diff
@@ -43,5 +43,10 @@ Feature: Gherkin terminal reporter
 
   Scenario: Should step parameters be replaced by their values
     Given there is gherkin scenario outline implemented
-    When tests are run with step expanded mode
+    When tests are run with step expanded option
+    Then output must contain parameters values
+
+  Scenario: Should step parameters be replaced by their values also when used together with gherkin reporter option
+    Given there is gherkin scenario outline implemented
+    When tests are run with step expanded and gherkin reporter options
```

Comment:
It would make it clearer to have a step like "when tests are run with --gherkin-terminal-reporter --gherkin-terminal-reporter-expanded". 
Also, these 2 tests are basically the same, the only thing that differ is the options. Should we instead parametrize the options in the scenario outline and have 1 scenario definition?

Reply:
I moved both tests into parameterized scenario :)